x-pack/plugins/uptime/public/apps/plugin.ts

```diff
@@ -84,7 +84,7 @@ export class UptimePlugin
     });
 
     core.application.register({
-      appRoute: '/app/uptime#/',
+      appRoute: '/app/uptime',
```

Comment:
this optional btw, platform will generate default route. So we can remove this.

Reply:
Thanks for pointing this out. I removed it https://github.com/elastic/kibana/pull/76421/commits/c3fa61bfe9ddb5be8e05eda5710c5a57990132bf.